### ISSUE TITLE
Valid Italian phone number fails plausibility check

### ISSUE THREAD

**Issue by davemcginnzendesk** [floere/phony#502]
Phone number +39 085 54105 fails plausibility, though it is a valid number:

![Screenshot 2022-07-27 at 08 09 55](https://user-images.githubusercontent.com/94527431/181184196-70529f10-4409-4cb0-9ad9-4df66a59f132.png)

What I did
----------
Added phone number to functional test:

```
      it_is_correct_for 'Italy', :samples => ['+39 0574 1234',
                                              '+39 085 54105',
                                              '+39 06 49911']
``` 

What happened
-------------
Test failed:

```
  1) plausibility plausible? specific countries is correct for Italy
     Failure/Error: Phony.plausible?(value).should be_truthy,
       It should validate +39 085 54105, but does not.
     # ./spec/functional/plausibility_spec.rb:28:in `block (3 levels) in it_is_correct_for'
     # ./spec/functional/plausibility_spec.rb:27:in `each'
     # ./spec/functional/plausibility_spec.rb:27:in `block (2 levels) in it_is_correct_for'
     # ./spec/functional/plausibility_spec.rb:21:in `each'
     # ./spec/functional/plausibility_spec.rb:21:in `block in it_is_correct_for'

Finished in 0.05001 seconds
122 examples, 1 failure
```

What I expected to happen and why
---------------------------------
Test should pass because the phone number is valid

**Comment by floere** [comment]
@davemcginnzendesk Thank you for the issue! 😊 I've released 2.19.15 which is more lenient towards Italian length 3 NDCs (more in accordance with https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200006B0001PDFE.pdf).

**Comment by floere** [comment]
Closing, let me know if there still is an issue, please.
